src/zenml/new/pipelines/pipeline.py

```diff
@@ -758,6 +754,10 @@ def _run(
                         "`zenml up`."
                     )
 
+            deploy_pipeline(
+                deployment=deployment_model, stack=stack, placeholder_run=run
+            )
+
```

Comment:
Why this reorder is needed?

Reply:
this is needed so that the URL for the pipeline is displayed before the pipeline run logs start.